src/test/groovy/io/github/cosmicsilence/scalafix/ScalafixPluginFunctionalTest.groovy

```diff
@@ -13,22 +14,22 @@ class ScalafixPluginFunctionalTest extends Specification {
     public final TemporaryFolder testProjectDir = new TemporaryFolder()
     private File settingsFile
     private File buildFile
+    private File scalaSrcFile
+    private File scalafixConfFile
 
     def setup() {
         settingsFile = testProjectDir.newFile("settings.gradle")
         buildFile = testProjectDir.newFile("build.gradle")
 
-        buildFile.write'''
-plugins {
-    id 'scala'
-    id 'io.github.cosmicsilence.scalafix'
-}
-
-repositories {
-    mavenCentral()
-}
-'''
+        buildFile.write ScalafixTestConstants.BUILD_SCRIPT
```

Comment:
let's keep the build script inlined here (instead of a constant) and append the other bits that you correctly introduced (Scala dependency + compiler option) 👍